pygmt/helpers/decorators.py

```diff
@@ -574,6 +565,14 @@ def new_module(*args, **kwargs):
                         f"Parameters in short-form ({short_param}) and "
                         f"long-form ({long_alias}) can't coexist."
                     )
+                if (long_alias in kwargs and long_alias in ["xshift", "yshift"]) or (
```

Comment:
I think it doesn't works, because after removing `X="xshift"` and `Y="yshift"` from the `use_alias` decorator, the variable `aliases` at line 571 no longer contain the paramters `xshift` or `yshift`.

Instead, after line 583, you can check if any of `xshift`, `yshift`, `X` and `Y` exists in `kwargs`. If yes, then raise an warning. For backward compatibility, you also need to do the "xshift"->X and "yshift"->"Y" conversions if "xshift" or "yshift" is used.

Reply:
So, you mean to leave `X="xshift" and Y="yshift"` as is in each function and add the others at this point, too @seisman ?
```
@use_alias(
    X="xshift",
    Y="yshift",
    xshift="X",
    yshift="Y",
)
```

Reply:
> So, you mean to leave `X="xshift" and Y="yshift"` as is in each function and add the others at this point, too @seisman ?
> 
> ```
> @use_alias(
>     X="xshift",
>     Y="yshift",
>     xshift="X",
>     yshift="Y",
> )
> ```

No. Changes to other files (e.g., `basemap.py`) make sense to me. The only thing that is missing is how to _correctly_ check if users use `xshift`, `yshift`, `X` or `Y`. The checking should be done in the `pygmt/helpers/decorators.py` file.